### PR TITLE
modify seq2seq example

### PR DESCRIPTION
**Pull request by kiyukuta**
fix some details

### CODE REVIEW

File: examples/seq2seq/seq2seq.py [hunk 1]
```diff
@@ -213,25 +219,26 @@ def main():
...
-                   trigger=(200, 'iteration'))
+    trainer = training.Trainer(updater, (args.epoch, 'epoch'), out=args.out)
+    trainer.extend(extensions.LogReport(trigger=trigger),
+                   trigger=trigger)
     trainer.extend(extensions.PrintReport(
         ['epoch', 'iteration', 'main/loss', 'validation/main/loss',
          'main/perp', 'validation/main/perp', 'validation/main/bleu',
          'elapsed_time']),
-        trigger=(200, 'iteration'))
+        trigger=trigger)
```
Comment (soskek) [inline examples/seq2seq/seq2seq.py]: (This is just my feeling, so you can leave this.)
I think it is user-friendly to group extensions into two, "small step" and "large step".
"Small step" group includes PrintReport, LogReport and sample generation. They and important for checking progress at a low cost. And also, it is useful just for seeing running without problems.
"Large step" group includes BLEU calculation and snapshot serialization. They take a cost to some extent. BLEU calculation over test dataset seem to take much time. Serialization needs a large storage size without user's awareness (if frequently).
So, what do you think about preparing two different step (small and large) sizes for triggers?

File: examples/seq2seq/seq2seq.py [hunk 2]
```diff
@@ -187,10 +191,12 @@ def main():
     else:
         # Check file
         en_path = os.path.join(args.input, 'giga-fren.release2.fixed.en')
-        source_vocab = ['<eos>', '<unk>'] + europal.count_words(en_path)
+        source_vocab = ['<eos>', '<unk>'] + europal.count_words(
+            en_path, limit=args.vocab)
```
Comment (kiyukuta) [inline examples/seq2seq/seq2seq.py]: oh, I forgot to change it. thanks!

File: examples/seq2seq/seq2seq.py (hunk 1 above)
Comment (kiyukuta) [inline examples/seq2seq/seq2seq.py]: I agree with you ! 
As your suggestion, I will make two triggers of different cycle.

**Comment by unnonouno** [comment]
I merged `_v2-seq2seq-europal` to `seq2seq-europal`. Please send this PR to `seq2seq-europal`.

**Pull request closed without merging**